convoy/slurm.py

```diff
@@ -552,7 +552,7 @@ def create_slurm_controller(
             over_subscribe = ' OverSubscribe={}'.format(part.over_subscribe)
         else:
             over_subscribe = ''
-        if util.is_not_empty(part.priority_tier):
+        if isinstance(part.priority_tier,int):
```

Comment:
`priority_tier` type is enforced in the yaml validator. Please change the check to:

`if part.priority_tier is not None:`